### PR TITLE
added certificate notes

### PR DESCRIPTION
**Pull request by Krishnan-Priyanshu**
##### SUMMARY

Certificate Resource added notes for ssl certificate content.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

#### Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note

```

#### Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

#### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Review by Sakshi-dell — DISMISSED** [review]

File: docs/resources/certificate.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ description: |-
 
 Resource for import the ssl certificate to iDRAC, on the basis of input parameter Type. After importing the certificate, the iDRAC will automatically restart.
 
+~> **Note:** We support two kinds of SSL certificates (1) Server certificate (2) Custom certificate 
+
+~> **Note:** Server Certificate: By default, the iDRAC comes with a self-signed certificate for its web server. If user wants to replace with her own server certificate (signed by Trusted CA). Steps:- (1) Generate the CSR from iDrac. (2) Create the certificate using CSR and sign with trusted CA. (3) The certificate should be signed with hashing algorithm equivalent to sha256
+
+~> **Note:** Custom Certificate: Steps:- (1) An externally created custom certificate which can be imported into the iDRAC. (2) Convert the external custom certificate into PKCS#12 format and should be encoded via base64. The conversation will required passphrase which should be supplied to 'passphrase' attribute during the import process."
```
Comment (rounak-adhikary) [inline docs/resources/certificate.md:33]:
```suggestion
~> **Note:** Custom Certificate: Steps:- (1) An externally created custom certificate which can be imported into the iDRAC. (2) Convert the external custom certificate into PKCS#12 format and should be encoded via base64. The conversion will require passphrase which should be provided in the `passphrase` attribute."
```

File: templates/resources/certificate.md.tmpl [hunk 2]
```diff
@@ -26,6 +26,14 @@ description: |-
 
 {{ .Description | trimspace }}
 
+~> **Note:** We support two kinds of SSL certificates (1) Server certificate (2) Custom certificate 
+
+~> **Note:** Server Certificate: By default, the iDRAC comes with a self-signed certificate for its web server. If user wants to replace with her own server certificate (signed by Trusted CA). Steps:- (1) Generate the CSR from iDrac. (2) Create the certificate using CSR and sign with trusted CA. (3) The certificate should be signed with hashing algorithm equivalent to sha256
+
+~> **Note:** Custom Certificate: Steps:- (1) An externally created custom certificate which can be imported into the iDRAC. (2) Convert the external custom certificate into PKCS#12 format and should be encoded via base64. The conversation will required passphrase which should be supplied to 'passphrase' attribute during the import process."
```
[the remark this answers is not in the document]
Comment (Krishnan-Priyanshu) [reply templates/resources/certificate.md.tmpl]: fixed doc

File: docs/resources/certificate.md (hunk 1 above)
Comment (Krishnan-Priyanshu) [reply docs/resources/certificate.md]: fixed doc

**Review by prajwalpatil25 — APPROVED** [review]

**Pull request merged**